kuwala/pipelines/population-density/src/Processor.py

```diff
@@ -46,7 +46,7 @@ def start(files: [dict], output_dir: str):
 
         df = reduce((lambda d1, d2: d1.join(d2, ['h3Index'], 'full').repartition(number_of_partitions, 'h3Index')), dfs)
 
-        df.write.mode('overwrite').parquet(output_dir + 'result.parquet')
+        df.write.mode('overwrite').parquet(output_dir + 'result'+'_'+updated_date+'.parquet')
```

Comment:
The name of the result file should follow the same structure as the downloaded files (`updated_date` + 'result' + `initial_date` + 'parquet').
<img width="350" alt="image" src="https://user-images.githubusercontent.com/28758040/147700806-79f17323-6c52-4cb1-8cd2-4a0c06531d19.png">

Reply:
ATTENTION: The database importer has to be adjusted accordingly here